ffi/diplomat/src/fixed_decimal.rs

```diff
@@ -28,24 +29,40 @@ pub mod ffi {
         Positive,
     }
 
+    /// ECMA-402 compatible sign display preference.
+    #[diplomat::rust_link(fixed_decimal::decimal::SignDisplay, Enum)]
+    pub enum ICU4XFixedDecimalSignDisplay {
+        Auto,
+        Never,
+        Always,
+        ExceptZero,
+        Negative,
+    }
+
     fn convert(dec: FixedDecimal) -> Box<ICU4XFixedDecimal> {
         Box::new(ICU4XFixedDecimal(dec))
     }
 
     impl ICU4XFixedDecimal {
         /// Construct an [`ICU4XFixedDecimal`] from an integer.
         #[diplomat::rust_link(fixed_decimal::decimal::FixedDecimal, Struct)]
-        pub fn create(v: i32) -> Box<ICU4XFixedDecimal> {
+        pub fn create_from_i64(v: i64) -> Box<ICU4XFixedDecimal> {
             Box::new(ICU4XFixedDecimal(FixedDecimal::from(v)))
         }
 
-        /// Construct an [`ICU4XFixedDecimal`] from an float, with enough digits to recover
-        /// the original floating point in IEEE 754 without needing trailing zeros
+        /// Construct an [`ICU4XFixedDecimal`] from an integer.
+        #[diplomat::rust_link(fixed_decimal::decimal::FixedDecimal, Struct)]
+        pub fn create_from_u64(v: u64) -> Box<ICU4XFixedDecimal> {
+            Box::new(ICU4XFixedDecimal(FixedDecimal::from(v)))
+        }
+
+        /// Construct an [`ICU4XFixedDecimal`] from an integer-valued float
         #[diplomat::rust_link(fixed_decimal::decimal::FixedDecimal::try_from_f64, FnInStruct)]
-        pub fn create_from_f64_with_max_precision(
+        #[diplomat::rust_link(fixed_decimal::decimal::DoublePrecision, Enum)]
```

Comment:
question (for myself when i come edit this later): in our new rust_link world, would this be hidden? Or just normal?

(Personally I find compact only necessary if you're going to have a ton of links)